internal/gatewayapi/backendtrafficpolicy.go

```diff
@@ -406,12 +401,19 @@ func (t *Translator) translateBackendTrafficPolicyForRoute(policy *egv1a1.Backen
 					if policy.Spec.UseClientProtocol != nil {
 						r.UseClientProtocol = policy.Spec.UseClientProtocol
 					}
+
+					if errs != nil {
```

Comment:
this needs to be done for all routes - tcp and udp too
and we need to move this to the beginning of the route loop so we
```
			for _, r := range http.Routes {
				// Apply if there is a match
				if strings.HasPrefix(r.Name, prefix) {
				  If errs != nil {
				   .....
				   continue
				  }
```

Reply:
Done 👍🏽